lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -279,9 +289,44 @@ int main(int argc, char* argv[]) {
   magma::SessionProxyResponderAsyncService proxy_service(
       server.GetNewCompletionQueue(), std::move(proxy_handler));
   server.AddServiceToServer(&local_service);
+  MLOG(MINFO) << "Add localservice";
   server.AddServiceToServer(&proxy_service);
+  MLOG(MINFO) << "Add proxyservice";
+  std::thread access_common_message_thread;
+  /* Initialize the main thread of session managment by folly event to handle
+   * logical component of 5G of SessionD
+   */
+  auto session_enforcer = std::make_shared<magma::SessionStateEnforcer>(
+                            rule_store, *session_store, pipelined_client,
+	                        amf_srv_client, mconfig);
+  //5G related async msg handler service framework creation
+  auto set_message_handler = std::make_unique<magma::SetMessageManagerHandler>(
+		  session_enforcer, *session_store);
+  MLOG(MINFO) << "session enforcer";
```

Comment:
Can this log be a bit more helpful? "Starting SetMessageManagerHandler" maybe?

Reply:
I would add a `5g` reference,

People reading the log may not know what `session enforcer` or `SetMessageManagerHandler` is, so they may not know if they are really running the 4g version or 5g version

Reply:
Changed